src/pages/stablecoins.tsx

```diff
@@ -0,0 +1,753 @@
+import { GetStaticProps } from "next/types"
+import { SSRConfig, useTranslation } from "next-i18next"
+import { serverSideTranslations } from "next-i18next/serverSideTranslations"
+import {
+  Box,
+  BoxProps,
+  Flex,
+  FlexProps,
+  Grid,
+  HeadingProps,
+  Icon,
+  SimpleGrid,
+} from "@chakra-ui/react"
+
+import ButtonLink from "@/components/Buttons/ButtonLink"
+import CalloutBanner from "@/components/CalloutBanner"
+import DataProductCard from "@/components/DataProductCard"
+import Emoji from "@/components/Emoji"
+import FeedbackCard from "@/components/FeedbackCard"
+import GhostCard from "@/components/GhostCard"
+import HorizontalCard from "@/components/HorizontalCard"
+import { Image } from "@/components/Image"
+import InfoBanner from "@/components/InfoBanner"
+import InlineLink from "@/components/Link"
+import OldHeading from "@/components/OldHeading"
+import Text from "@/components/OldText"
+import PageHero from "@/components/PageHero"
+import PageMetadata from "@/components/PageMetadata"
+import ProductList from "@/components/ProductList"
+import StablecoinAccordion from "@/components/StablecoinAccordion"
+import StablecoinBoxGrid from "@/components/StablecoinBoxGrid"
+import StablecoinsTable from "@/components/StablecoinsTable"
+import Tooltip from "@/components/Tooltip"
+
+import { getLastDeployDate } from "@/lib/utils/getLastDeployDate"
+import { runOnlyOnce } from "@/lib/utils/runOnlyOnce"
+import { getRequiredNamespacesForPage } from "@/lib/utils/translations"
+
+import {
+  BASE_TIME_UNIT,
+  COINGECKO_API_BASE_URL,
+  COINGECKO_API_URL_PARAMS,
+} from "@/lib/constants"
+
+import dogeComputerImg from "@/public/doge-computer.png"
+// -- daps
+import aaveImg from "@/public/stablecoins/aave.png"
+import compoundImg from "@/public/stablecoins/compound.png"
+// Static assets
+import daiLargeImg from "@/public/stablecoins/dai-large.png"
+import heroImg from "@/public/stablecoins/hero.png"
+import stablecoinsWtfImg from "@/public/stablecoins/tools/stablecoinswtf.png"
+import usdcLargeImg from "@/public/stablecoins/usdc-large.png"
+
+type EthereumDataResponse = Array<{
+  id: string
+  name: string
+  market_cap: number
+  image: string
+  symbol: string
+}>
+
+type StablecoinDataResponse = Array<{
+  id: string
+  name: string
+  market_cap: number
+  image: string
+  symbol: string
+}>
+
+interface Market {
+  name: string
+  marketCap: string
+  image: string
+  type: string
+  url: string
+}
+
+type Props = SSRConfig & {
+  lastDeployDate: string
+}
+
+const ethereumDataFetch = runOnlyOnce(() =>
+  fetch(
+    `${COINGECKO_API_BASE_URL}ethereum-ecosystem${COINGECKO_API_URL_PARAMS}`
+  ).then((res) => res.json())
+)
```

Comment:
Use `async/await` for consistency. Another improvement would be to move this function to `@/lib/api` as we are doing in other cases, to have all the api calls in one place.
```suggestion
const ethereumDataFetch = runOnlyOnce(async () => {
  {
    const res = await fetch(
      `${COINGECKO_API_BASE_URL}ethereum-ecosystem${COINGECKO_API_URL_PARAMS}`
    )
    return res.json()
  }
})
```

Reply:
Fixed